pkg/joinserver/config.go

```diff
@@ -24,4 +24,5 @@ type Config struct {
 	JoinEUIPrefixes               []types.EUI64Prefix                  `name:"join-eui-prefix" description:"JoinEUI prefixes handled by this Join Server"`
 	DefaultJoinEUI                types.EUI64                          `name:"default-join-eui" description:"Default JoinEUI for this Join Server"`
 	DeviceKEKLabel                string                               `name:"device-kek-label" description:"Label of KEK used to encrypt device keys at rest"`
+	DevNonceLimit                 int                                  `name:"dev-nonce-limit" description:"Sets the amount of the DevNonces to be stored per device"`
```

Comment:
Consistency: `Amount of DevNonces stored per device` - the description is for the field, not for the configuration that you see with `--help`